tests/test_tree_tests.sh

```diff
@@ -25,39 +26,17 @@ cp -r "${TOP}" "${SRCDIR}/blueprint"
 
 cd "${OUTDIR}"
 export BLUEPRINTDIR=${SRCDIR}/blueprint
+#setup
 ${SRCDIR}/blueprint/bootstrap.bash $@
+
+#confirm that a rebuild happens when a glob match is added
 ./blueprint.bash
 
 OLDTIME_BOOTSTRAP=$(mtime .bootstrap/build.ninja)
 OLDTIME=$(mtime build.ninja)
 
-sleep 2
-./blueprint.bash
-
-if [ ${OLDTIME} != $(mtime build.ninja) ]; then
-    echo "unnecessary build.ninja regeneration for null build" >&2
-    exit 1
-fi
```

Comment:
Why are these tests being removed?

Reply:
they didn't work in this patchset, but the nextpatchset fixes them and keeps them in